app/Filament/Resources/NewsResource.php

```diff
@@ -0,0 +1,149 @@
+use App\Filament\Resources\NewsResource\Pages;
+use App\Models\News;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\HtmlString;
+
+class NewsResource extends Resource
+{
+    protected static ?string $model = News::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-newspaper';
+
+    protected static ?string $navigationGroup = 'Community';
+
+    protected static ?int $navigationSort = 1;
+
+    protected static ?string $recordTitleAttribute = 'Title';
+
+    protected static int $globalSearchResultsLimit = 2;
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Forms\Components\Section::make('Primary Details')
+                    ->icon('heroicon-m-key')
+                    ->columns(['md' => 2])
+                    ->schema([
+                        Forms\Components\TextInput::make('Title')
+                            ->required()
+                            ->minLength(2)
+                            ->maxLength(60),
+
+                        Forms\Components\TextInput::make('Link')
+                            ->label('URL')
+                            ->required()
+                            ->activeUrl(),
+                    ]),
+
+                Forms\Components\Section::make('Leading Text')
+                    ->icon('heroicon-m-chat-bubble-oval-left-ellipsis')
+                    ->description("
+                        This short description will appear on the site's home page and in any news feed features we support.
+                        This should be a VERY short plain-text summary of what the user can find at the URL. Anticipate that
+                        users will only spend 5 seconds or less to read this, so be very concise!
+                    ")
+                    ->schema([
+                        Forms\Components\Textarea::make('Payload')
+                            ->label('Lead')
+                            ->maxLength(238) // Have some guardrails so people don't go crazy with these.
+                            ->helperText(function (?string $state, Forms\Components\Textarea $component) {
+                                $maxLength = $component->getMaxLength();
+
+                                return new HtmlString('
+                                    <div class="flex w-full justify-between">' .
+                                        '<p>HTML is not supported.</p>' .
+
+                                        '<p>' .
+                                            strlen($state ?? '') . '/' . $maxLength .
+                                        '</p>' .
+                                    '</div>'
+                                );
+                            })
+                            ->reactive()
+                            ->required(),
```

Comment:
Is there some way to make this box taller? It seems to default to two lines.

Reply:
Bumped to 4 in latest.